news/2022-12-01-osumonthly-5.md

```diff
@@ -0,0 +1,184 @@
+
+Nowadays, player skill far surpasses what the Ranked section has to offer. But who knows — maybe the next, greatest challenge will finally put players at bay.
+
+## Did you know?
+
+There are many [user groups](/wiki/People/The_Team#user-groups) on osu!, most often recognizable by their title or color, indicating their membership in that group.
+
+During this month, a new group has appeared on the osu! website. Indeed, the [Beatmap Spotlight Curators team](https://osu.ppy.sh/groups/48) has received their own group as a means to distinguish them from regular users and provide more transparency as to who is involved with the project.
+
+Furthermore, descriptions have been added to all user groups which has also contributed to transparency and informing the players as to what the purpose of said group is, all in all a very good addition to the website.
+
+## Monthly community quiz
+
+Last month's question took a bit longer to answer than usual, but the answer was still found!
+
+Congratulations to [metis_](https://osu.ppy.sh/users/8171561) for being the first person to answer last month's question and score themselves 1 month of osu!supporter!
+For anyone who missed last month's edition, the question was:
+
+*"What symbol of Halloween went extinct in early 2015, and what did they drop after their reincarnation?"*
+
+and the answer was: *BAT (Beatmap Appreciation Team)*. If you were curious, the "T" was dropped after the group was renamed into "Beatmap Appreciators".
+
+Thanks to chromb we have got yet another puzzling question for you, and that question is...:
+
+TBA 
+
+The rules are as usual, you have 1 month since the news post release to answer, first person to answer correctly receives 1 month of osu!supporter. Good luck!
+
+## Things to look out for in December
+
+- A new LAN is organized next month, but this time, in France! The [UTT Arena](https://osu.ppy.sh/community/forums/topics/1661444?n=1) organized tournaments for osu! and osu!mania, so if you are interested in the French scene, be sure to catch the top French players compete on the hardest maps by tuning into their livestream on [Twitch](https://www.twitch.tv/osufrlive).
+- The "New Beginnings" art contest [has been announced and is underway](https://osu.ppy.sh/community/contests/160)! You have until mid-December to submit your wonderful artworks, or if you are not the creative or drawing type, sit tight as we await for the voting to begin just around the winter holidays!
+
+## End word
+
+Thank you for reading this month's edition of osu!monthly! Next month we will be making some announcements in regards to additional content being added onto osu!monthly and we would love to hear your opinions! As always, if you have any words of encouragement or criticism, we would also love for you to reach out and let us know what we can do better.
+
+The osu!monthly team once again wishes you the best during the current holiday season!
+
+—0x84f, Junihuhn, Pisapou
```

Comment:
why aren't all co-writers listed here 🤔

Reply:
I was worried that we'd have too many co-writers so I opted to have people who are doing "smaller" segments such as plays for the individual game modes be credited at the top of their segment. If you think we can have all of them written in the authors mark I don't have anything against it

Reply:
Quick doubt then. Didn't see mania's play be credited, nor was I here. Unsure if that should be mentioned somewhere (either in the Play of the Month, on Silly's section, both, or at the end here).

Reply:
well if you have like 10 people writing small sections then sure, use your own discretion. i just brought this up because iirc the Mappers' Report credited everyone here.

and it's not like github reviewers get credited either ([most of the time](https://osu.ppy.sh/home/news/2021-11-09-performance-points-star-rating-updates)) so i guess this is up for debate anyway

Reply:
@MChecaH shoot yeah you were not credited, I will fix that (and since you also wrote Silly's part I will add you to co-authors), but there needs to be a discussion about possibly having 1 name for author's mark and whether everyone's parts will be individually credited like the plays

Reply:
Fixed that in [3fa490e](https://github.com/ppy/osu-wiki/pull/8416/commits/3fa490e4ca3dfa5e4323e0b4d3bbae5e8b573e92), I will start a discussion about coauthoring after we get this release sorted